cfpb/cfgov-sheer-templates
# Updates ondemand header/footer/secondary-nav to CFv4

Re-generates header/footer/secnav on-demand CSS from cfgov-refresh to include CFv4 changes.

To test: 
- Copy these changes into local owning-a-home node_modules/cfgov-sheer-templates
- Follow running OAH with https://github.com/cfpb/cfgov-refresh/blob/master/docs/development-tips.md#tip-loading-sibling-projects
- Check /owning-a-home/loan-options/FHA-loans/ that the header is the right width etc. vs the live site.
